microraiden/microraiden/test/fixtures/client.py

```diff
@@ -62,24 +64,26 @@ def client(
         sender_privkey: str,
         client_contract_proxy: ChannelContractProxy,
         client_token_proxy: ContractProxy,
-        datadir: str
+        datadir: str,
+        verbose: bool,
+        clean_channels: bool,
+        receiver_privkey: str
```

Comment:
I understand why you need this here, but isn't there a better place to do a channel cleanup? i.e. when tearing down `ChannelManager`?

Reply:
There are some client tests that run without a channel manager. Are there channel manager tests that open channels without this client fixture?